Files/Interacts/Interaction.cs

```diff
@@ -1329,17 +1266,17 @@ private void NewFile(ShellNewEntry itemType)
             CreateFileFromDialogResultType(AddItemType.File, itemType);
         }
 
-        public RelayCommand SelectAllContentPageItems => new RelayCommand(() => SelectAllItems());
+        public RelayCommand SelectAllContentPageItems => new RelayCommand(() => SelectAllItems(AssociatedInstance.ContentPage));
```

Comment:
I'd suspect @yaichenbaum has some good opinions about where we should place the RelayCommands for code quality.

Reply:
@duke7553 I would make separate classes to group relay commands depending on the kind of actions. For example, clipboard actions could go be grouped together, selection commands could be placed in a separate class etc.